README.md

```diff
@@ -254,42 +254,16 @@ provides(Sources,URI("http://libvirt.org/sources/libvirt-1.1.1-rc2.tar.gz"),libv
 
 # The high level interface - Loading dependencies
 
-BinDeps provides the `@BinDeps.load_dependencies` macro that you may call early in 
-initialization process of your package to load all declared libraries in your build.jl
-file. 
-
-The basic usage is very simple:
-
-```jl
-using BinDeps
-@BinDeps.load_dependencies
-```
-
-This will make all your libraries available as variables named by the names you gave 
-the dependency. E.g. if you declared a dependency as
-
+To load dependencies without a runtime dependence on BinDeps, place code like the following near the start of the module's primary file. Don't forget to change the error message to reflect the name of your package.
 ```julia
-	library_dependency("libfoo")
+const depsfile = joinpath(dirname(@__DIR__), "deps", "deps.jl")
```

Comment:
it is one level different though, so would need another dirname or another `"..",` argument